### PR TITLE
[spi-hdlc-adapter] add delay after reset the RCP chip

### PR DESCRIPTION
**Pull request by zhanglongxia**

### CODE REVIEW

File: tools/spi-hdlc-adapter/spi-hdlc-adapter.c [hunk 1]
```diff
@@ -1632,6 +1636,18 @@ int main(int argc, char *argv[])
                 syslog(LOG_NOTICE, "SPI CS Delay set to %d usec", sSpiCsDelay);
                 break;
 
+            case ARG_SPI_RESET_DELAY:
+                assert(optarg);
+
+                sSpiResetDelay = atoi(optarg);
+                if (sSpiResetDelay < 0)
+                {
+                    syslog(LOG_ERR, "Negative values (%d) for --spi-reset-delay are invalid.", sSpiResetDelay);
```
Comment (bukepo) [inline tools/spi-hdlc-adapter/spi-hdlc-adapter.c]: Negative value (%d) for --spi-reset-delay us invalid.

Comment (zhanglongxia) [reply tools/spi-hdlc-adapter/spi-hdlc-adapter.c]: Updated.

File: tools/spi-hdlc-adapter/spi-hdlc-adapter.c [hunk 2]
```diff
@@ -154,9 +154,10 @@ static int sIntGpioValueFd = -1;
 static int sHdlcInputFd  = -1;
 static int sHdlcOutputFd = -1;
 
-static int     sSpiSpeed   = 1000000; // in Hz (default: 1MHz)
-static uint8_t sSpiMode    = 0;
-static int     sSpiCsDelay = 20; // in microseconds
+static int     sSpiSpeed      = 1000000; // in Hz (default: 1MHz)
+static uint8_t sSpiMode       = 0;
+static int     sSpiCsDelay    = 20; // in microseconds
+static int     sSpiResetDelay = 1;  // in milliseconds
```
Comment (abtink) [inline tools/spi-hdlc-adapter/spi-hdlc-adapter.c]: Suggest using zero as default the keep behavior same as before.

Comment (zhanglongxia) [reply tools/spi-hdlc-adapter/spi-hdlc-adapter.c]: Updated

**Pull request merged**